de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/joiner/JoinerNodeModel.java

```diff
@@ -834,17 +840,31 @@ private static String getObjectAsJSONString(Object object) {
   private static String[] getFSKObjectAsStringArray(FskPortObject portObject) {
     return new String[] {getObjectAsJSONString(portObject.modelMetadata),
         getObjectAsJSONString(portObject.getModel()), getObjectAsJSONString(portObject.getViz()),
-        getObjectAsJSONString(portObject.simulations), getObjectAsJSONString(portObject.packages)};
-  }
+        getObjectAsJSONString(portObject.simulations), getObjectAsJSONString(portObject.packages),
+        ""};
+  }
+
+  private static FskPortObject getFSKObjectFromStringArray(Optional<EnvironmentManager> manager,
+      String[] model, String modelType)
+      throws JsonMappingException, JsonProcessingException, IOException {
```

Comment:
JsonMappingException and JsonProcessingException are subclasses of IOException and not needed.